src/components/DropDown/index.tsx

```diff
@@ -0,0 +1,25 @@
+'use client';
+
+import * as S from './style';
+
+const DropDown = ({}) => {
```

Comment:
이 부분에 중괄호는 제거하셔도 될것 같습니다
```suggestion
const DropDown = () => {
```

Reply:
불필요한 중괄호 제거 했습니다.